### PR TITLE
Bump Helm chart to v1.28.0

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Bump Helm chart to v1.28.0

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: helm/cloud-provider-azure/templates/cloud-provider-azure.yaml [hunk 1]
```diff
@@ -7,11 +7,13 @@
...
-  {{- $_ := set . "cloudProviderAzureVersion" "v1.25.17" }}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.25.18" }}
 {{- else if eq .Capabilities.KubeVersion.Minor "26" -}}
-  {{- $_ := set . "cloudProviderAzureVersion" "v1.26.13" }}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.26.14" }}
 {{- else if eq .Capabilities.KubeVersion.Minor "27" -}}
-  {{- $_ := set . "cloudProviderAzureVersion" "v1.27.7" }}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.27.8" }}
+{{- else if eq .Capabilities.KubeVersion.Minor "28" -}}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.28.0" }}
```
Comment (CecileRobertMichon) [inline helm/cloud-provider-azure/templates/cloud-provider-azure.yaml:16]: I verified that all images referenced here are published

**Review by mboersma — COMMENTED** [review]
/lgtm

(Looks like README.md has already been updated.)

**Comment by lzhecheng** [comment]
/approve
Thank you!

**Pull request merged**